tests/generators/test_python.py

```diff
@@ -448,6 +448,22 @@ def test_when_property_not_set_then_returns_None(
 
         assert getter() is None
 
+    def test_when_get_property_on_blank_TraitsData_view_then_returns_None(
```

Comment:
[consider][strong] Be a little clearer about what exactly the when is, and match the convention for the other cases (same below):
```suggestion
    def test_when_trait_not_set_then_returns_None(
```

Reply:
Sure, was trying to mirror the given/when wording from the cpp test, but had to munge it a fair bit to get the longer default one to fit inside the line length. Have taken your suggestion instead. 
https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/63/commits/ccdeea88fde9b4ba90c1105bc93b40aa5dbfdc32

Reply:
Thanks - for some reason that diff is mind boggling, but the end result looks 👌